2021-09-15 14:26:07,797 [INFO] [ai.py] Initialazing...
2021-09-15 14:26:07,814 [INFO] [ai.py] Importing DLL(s)...
2021-09-15 14:26:07,845 [INFO] [ai.py] Import complete.
2021-09-15 14:26:07,846 [INFO] [ai.py] Loading the network...
2021-09-15 14:26:08,162 [INFO] [ai.py] [class_colors] names is: <class 'list'>
2021-09-15 14:26:08,164 [INFO] [ai.py] Network loaded.
2021-09-15 14:26:08,165 [INFO] [ai.py] Initialazing complete.
2021-09-15 14:26:08,271 [INFO] [ai.py] Updating AI.
2021-09-15 14:26:08,273 [INFO] [ai.py] [image_detection] image is: <class 'NoneType'>, network is: <class 'int'>, class_names is: <class 'list'> class_colors is: <class 'dict'>, thresh is: <class 'float'>
2021-09-15 14:26:08,276 [INFO] [ai.py] [network_width] net is: <class 'int'>
2021-09-15 14:26:08,277 [INFO] [ai.py] [network_height] net is: <class 'int'>
2021-09-15 14:26:08,291 [WARNING] [ai.py] Something went wrong while updating.
2021-09-15 14:26:08,293 [WARNING] [ai.py] OpenCV(4.5.3) C:\Users\runneradmin\AppData\Local\Temp\pip-req-build-sn_xpupm\opencv\modules\imgproc\src\color.cpp:182: error: (-215:Assertion failed) !_src.empty() in function 'cv::cvtColor'

2021-09-15 14:26:08,295 [INFO] [ai.py] Update complete.
2021-09-15 14:27:01,997 [INFO] [ai.py] Initialazing...
2021-09-15 14:27:01,997 [INFO] [ai.py] Initialazing...
2021-09-15 14:27:01,997 [INFO] [ai.py] Importing DLL(s)...
2021-09-15 14:27:01,997 [INFO] [ai.py] Importing DLL(s)...
2021-09-15 14:27:02,000 [INFO] [ai.py] Import complete.
2021-09-15 14:27:02,000 [INFO] [ai.py] Import complete.
2021-09-15 14:27:02,000 [INFO] [ai.py] Loading the network...
2021-09-15 14:27:02,000 [INFO] [ai.py] Loading the network...
2021-09-15 14:27:02,289 [INFO] [ai.py] [class_colors] names is: <class 'list'>
2021-09-15 14:27:02,289 [INFO] [ai.py] [class_colors] names is: <class 'list'>
2021-09-15 14:27:02,290 [INFO] [ai.py] Network loaded.
2021-09-15 14:27:02,290 [INFO] [ai.py] Network loaded.
2021-09-15 14:27:02,291 [INFO] [ai.py] Initialazing complete.
2021-09-15 14:27:02,291 [INFO] [ai.py] Initialazing complete.
2021-09-15 14:27:02,393 [INFO] [ai.py] Updating AI.
2021-09-15 14:27:02,393 [INFO] [ai.py] Updating AI.
2021-09-15 14:27:02,395 [INFO] [ai.py] [image_detection] image is: <class 'NoneType'>, network is: <class 'int'>, class_names is: <class 'list'> class_colors is: <class 'dict'>, thresh is: <class 'float'>
2021-09-15 14:27:02,395 [INFO] [ai.py] [image_detection] image is: <class 'NoneType'>, network is: <class 'int'>, class_names is: <class 'list'> class_colors is: <class 'dict'>, thresh is: <class 'float'>
2021-09-15 14:27:02,397 [INFO] [ai.py] [network_width] net is: <class 'int'>
2021-09-15 14:27:02,397 [INFO] [ai.py] [network_width] net is: <class 'int'>
2021-09-15 14:27:02,399 [INFO] [ai.py] [network_height] net is: <class 'int'>
2021-09-15 14:27:02,399 [INFO] [ai.py] [network_height] net is: <class 'int'>
2021-09-15 14:27:02,402 [WARNING] [ai.py] Something went wrong while updating.
2021-09-15 14:27:02,402 [WARNING] [ai.py] Something went wrong while updating.
2021-09-15 14:27:02,404 [WARNING] [ai.py] OpenCV(4.5.3) C:\Users\runneradmin\AppData\Local\Temp\pip-req-build-sn_xpupm\opencv\modules\imgproc\src\color.cpp:182: error: (-215:Assertion failed) !_src.empty() in function 'cv::cvtColor'

2021-09-15 14:27:02,404 [WARNING] [ai.py] OpenCV(4.5.3) C:\Users\runneradmin\AppData\Local\Temp\pip-req-build-sn_xpupm\opencv\modules\imgproc\src\color.cpp:182: error: (-215:Assertion failed) !_src.empty() in function 'cv::cvtColor'

2021-09-15 14:27:02,409 [INFO] [ai.py] Update complete.
2021-09-15 14:27:02,409 [INFO] [ai.py] Update complete.
